Bottom: a53de8f238f7910f979bb53f53f025f799110f93
Top:    7dce4c415b86085b4718701aa45fc9f061036ab1
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2013-02-04 17:09:15 -0800

Refresh of rstream-cnt

---

diff --git a/examples/common.c b/examples/common.c
index 28c0fb6..4eb5bbe 100644
--- a/examples/common.c
+++ b/examples/common.c
@@ -121,7 +121,7 @@ int size_to_count(int size)
 		return 100;
 	else if (size >= (1 << 16))
 		return 1000;
-	else if (size >= (1 << 11))
+	else if (size >= (1 << 10))
 		return 10000;
 	else
 		return 100000;
